packages/big-design/src/components/Counter/private.ts

```diff
@@ -0,0 +1,19 @@
+import { remCalc, theme as defaultTheme } from '@bigcommerce/big-design-theme';
```

Comment:
Is there a reason why this file is called `private.ts` and not `styled.ts`?

Reply:
I misunderstood advice on how to best implement this 🙅